src/usdb_syncer/settings.py

```diff
@@ -71,10 +71,11 @@ class SettingKey(Enum):
     TXT = "downloads/txt"
     ENCODING = "downloads/encoding"
     NEWLINE = "downloads/newline"
+    FORMAT_VERSION = "downloads/format_version"
     FIX_LINEBREAKS = "fixes/linebreaks"
-    FIX_FIRST_WORDS_CAPITALIZATION = "fixes/firstwordscapitalization"
+    FIX_FIRST_WORDS_CAPITALIZATION = "fixes/first_words_capitalization"
```

Comment:
Every time you change the key, the user stored settings gets lost.

Reply:
I was aware of that, but since it was only recently introduced and I noticed that this key does not follow the other key spellings, I decided to change it. But I can change that back to the way it was if you prefer.